tests/test_pyxdl.py

```diff
@@ -21,6 +21,9 @@ def test_damerau_levenshtein_distance(self):
         assert damerau_levenshtein_distance('orange', 'pumpkin') == 7
         assert damerau_levenshtein_distance('gifts', 'profit') == 5
         assert damerau_levenshtein_distance('Sjöstedt', 'Sjostedt') == 1
+        assert damerau_levenshtein_distance([1, 2, 3], [1, 3, 2]) == 1
+        assert damerau_levenshtein_distance((1, 2, 3), (1, 3, 2)) == 1
+        assert damerau_levenshtein_distance((1, 2, 3), [1, 3, 2]) == 1
```

Comment:
Could we possibly add a couple more slightly more complex tests here? For example, test tuples and also test for a couple non-1 results.